src/api/FileCollaborators.js

```diff
@@ -87,6 +88,36 @@ class FileCollaborators extends MarkerBasedAPI {
             requestData,
         });
     }
+
+    /**
+     * Fetches file @mention's
+     *
+     * @oaram {string} fileId
+     * @param {Function} successCallback
+     * @param {Function} errorCallback
+     * @param {string} searchStr - Search string to filter file collaborators by
+     * @param {Object} [options]
+     * @param {boolean} [options.includeGroups] - return groups as well as users
+     * @return {void}
+     */
+    getCollaboratorsWithQuery = (
+        fileId: string,
+        successCallback: Collaborators => void,
```

Comment:
Should this share the same type as the success callback in `getFileCollaborators`?